ilc_scrape.py

```diff
@@ -273,7 +274,7 @@ def main():
         print_quit("No lectures found. Is the url proper?")
 
     total_lecs = len(lectures)
-    subject_name = "{subjectName} {sessionName}".format(**lectures[0])
+    subject_name = sanitize_filename("{subjectName} {sessionName}".format(**lectures[0]))
```

Comment:
The line length should be less than 88 characters. Follow PEP8 convention.
```suggestion
    subject_name = sanitize_filename(
        "{subjectName} {sessionName}".format(**lectures[0])
    )
```